test/functional/page_objects/visualize_page.js

```diff
@@ -902,6 +917,9 @@ export function VisualizePageProvider({ getService, getPageObjects }) {
       const getChartTypesPromises = chartTypes.map(async chart => await chart.getAttribute('data-label'));
       return await Promise.all(getChartTypesPromises);
     }
+    async hasPieChartError() {
```

Comment:
This makes it sound like it will return true or false, but it's using `existOrFail` which will throw an error if it doesn't exist.  I'd remove the `return` and rename the function something like `assertHasPieChartError` . What do you think?

Reply:
Yes, it's for sure a better and meaningful name 👍